libs/community/langchain_community/llms/watsonxllm.py

```diff
@@ -287,9 +291,6 @@ def _stream_response_to_generation_chunk(
             generation_info=dict(
                 finish_reason=stream_response["results"][0].get("stop_reason", None),
                 llm_output={
-                    "generated_token_count": stream_response["results"][0].get(
```

Comment:
Was this creating failures? Why is this removed?

Reply:
@efriis  I was getting error `TypeError: Additional kwargs key generated_token_count already exists in left dict and value has unsupported type <class 'int'>.` when i used:
```
for a in watsonx_llm.stream("Write poem to sun"):
    print(a, end='')
```
or 
```
for a in watsonx_llm.generate(prompts=["Write poem to sun"], stream=True):
    print(a)
```
The error is related with `merge_dicts` method in `langchain_core.utils._merge`